components/tools/OmeroWeb/test/integration/test_metadata.py

```diff
@@ -49,9 +49,10 @@ def test_pixel_size_units(self):
         request_url = reverse('load_metadata_details', args=['image', iid])
         data = {}
         rsp = get(self.django_client, request_url, data, status_code=200)
-        html = rsp.content
+        html = rsp.content.decode("utf-8")
         # Units are µm by default
-        assert "Pixels Size (XYZ) (µm):" in html
+        value = "Pixels Size (XYZ) ("+u"µm):"
```

Comment:
Did you try these the other way around, i.e. keeping html as bytes and making `value` bytes as well?

Reply:
I can't remember since I have been juggling with many of them. I can give it a shot

Reply:
Now I remembered. 
I tried it before
`` UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 30979: ordinal not in range(128)``
That's the error I got at the time

Reply:
Understood. I imagine when we go py3-only (and add mypy annotations) we will discover that we can keep decode less.